CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# Contribution Guidelines
+
+## Table of Contents
+
+- [Contribution Guidelines](#contribution-guidelines)
+  - [Introduction](#introduction)
+  - [Bug reports](#bug-reports)
+  - [Discuss your design](#discuss-your-design)
+  - [Testing](#testing)
+  - [Translation](#translation)
+  - [Code review](#code-review)
+  - [Styleguide](#styleguide)
+  - [Design guideline](#design-guideline)
+  - [Developer Certificate of Origin (DCO)](#developer-certificate-of-origin-dco)
+  - [Release Cycle](#release-cycle)
+  - [Maintainers](#maintainers)
+  - [Owners](#owners)
+  - [Versions](#versions)
+  - [Releasing Total Open Station](#releasing-total-open-station)
+  - [Copyright](#copyright)
+
+## Introduction
+
+This document explains how to contribute changes to the Total Open Station project.
+It assumes you have followed the
+[installation instructions](https://totalopenstation.readthedocs.io/).
+
+## Bug reports
+
+Please search the issues on the issue tracker with a variety of keywords
+to ensure your bug is not already reported.
+
+If unique, [open an issue](https://github.com/steko/totalopenstation/issues/new)
+and answer the questions so we can understand and reproduce the
+problematic behavior.
+
+To show us that the issue you are having is in Total Open Station itself, please
+write clear, concise instructions so we can reproduce the behavior—
+even if it seems obvious. The more detailed and specific you are,
+the faster we can fix the issue. Check out [How to Report Bugs
+Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html).
+
+Please be kind, remember that Total Open Station comes at no cost to you, and you're
+getting free help.
+
+## Discuss your design
+
+The project welcomes submissions. If you want to change or add something,
+please let everyone know what you're working on—[file an issue](https://github.com/steko/totalopenstation/issues/new)!
+Significant changes must go through the change proposal process
+before they can be accepted. To create a proposal, file an issue with
+your proposed changes documented, and make sure to note in the title
+of the issue that it is a proposal.
+
+This process gives everyone a chance to validate the design, helps
+prevent duplication of effort, and ensures that the idea fits inside
+the goals for the project and tools. It also checks that the design is
+sound before code is written; the code review tool is not the place for
+high-level discussions.
+
+## Testing
+
+Before submitting a pull request, run all the tests for the whole tree
+to make sure your changes don't cause regression elsewhere.
+
+Here's how to run the test suite:
+
+- Install pytest
+- Run pytest from the root of the source tree
+
+## Translation
+
+We do all translation work inside [Transifex](https://www.transifex.com/projects/p/totalopenstation/resource/totalopenstation-app/). Once a translation has reached
+A SATISFACTORY PERCENTAGE it will be synced back into this repo and
```

Comment:
Hahahaha "A SATISFACTORY PERCENTAGE" isn't it to vague ?

Reply:
Yes, this is just copy and paste from the gitea docs.. we can just say 100% since the strings are so few